sections/multicolumn.liquid

```diff
@@ -167,6 +167,15 @@
       "default": "adapt",
       "label": "t:sections.multicolumn.settings.image_ratio.label"
     },
+    {
+      "type": "range",
+      "id": "columns_to_show_row",
+      "min": 1,
+      "max": 6,
```

Comment:
I removed the logic from the extra margin on desktop and kept it on tablet. Let me know your thoughts 🤔  https://github.com/Shopify/dawn/pull/989/commits/84123722342fb13549038c94593a210c24f56747

Reply:
Oh I see what you mean now. @wiktoriaswiecicka @melissaperreault what do you think?

1) Allow the spacing between columns to be affected by the grid desktop vertical spacing global setting
2) Dont allow it

Reply:
I think it's good and should stay the way it is. So that global settings impacts the spacing around the column from the multi columns section. 

We just need to remove that `margin-top` applied to them on tablet/mobile.